pages/blog/_slug.vue

```diff
@@ -171,6 +171,8 @@ export default {
         type: 'article',
         url: `https://formester.com/blog/${this.$route.params.slug}/`,
         title: this.article.title,
+        metaTitle: this.article.metaTitle,
+        metaDescription: this.article.metaDescription,
         description: this.article.description,
```

Comment:
I think this should be title and description instead of metaTitle and metaDescription can you check this
```suggestion
        description: this.article.metaDescription,
```